docs/src/integrations/observability/new-relic/php.md

```diff
@@ -2,11 +2,8 @@
 title: "PHP"
 ---
 
-{{< note title="PHP version">}}
-
-Due to New Relic [dropping support for Zend Thread Safe (ZTS) versions](https://docs.newrelic.com/docs/release-notes/agent-release-notes/php-release-notes/php-agent-9170300/), you can't use New Relic with PHP v8.0+. To use New Relic with PHP, specify `type: 'php:7.4'` in your `.platform.app.yaml` file.
-
-{{< /note >}}
+New Relic isn't yet available on PHP 8.1.
+To use New Relic with PHP, specify `type: 'php:8.0'` or lower in your `.platform.app.yaml` file.
```

Comment:
[Explaining how to change the runtime versions might also be nice?](https://pr-2174-in2qtpq-652soceglkw4u.eu-3.platformsh.site/configuration/app/app-reference.html#example-configuration)
Since we describe a change that has to be done in a file.

Reply:
I'm not sure what you mean by explaining how to change that. I could put in a link to the app reference like so:
```suggestion
To use New Relic with PHP, specify `type: 'php:8.0'` or lower in your [app configuration](../../../configuration/app/app-reference.md#type).
```
Would that do it? I don't see the steps that would work in a how to, but I might be missing something.

Reply:
Yes, that's exactly what I meant 😎 To make the change super easy for newcomers.